.gitignore

```diff
@@ -86,3 +86,6 @@ fastlane/test_output
 
 # Sonarqube
 .scannerwork
+
+# Vi
+.swp
```

Comment:
Isn't this file generated anyway? I bet they support vim too.

Reply:
Alternatively, does it make sense to commit this at all? Would we accept a PR for emacs support in our gitignore?

Maybe this is something users should configure locally, like so? https://stackoverflow.com/a/5724476

Reply:
Since its mostly me working on this code I quickly setup the gitignore to fit my working environment. User level global gitignores would be a good way of solving this as well. Probably superior even. 👍  

If someone regularly commits to the project and wants an emacs gitignore I'd be happy to accept a PR. 😅

Reply:
Fair enough